server/src/main/scala/sbt/server/SbtDiscovery.scala

```diff
@@ -48,10 +48,10 @@ private[server] object SbtDiscovery {
   // NOTE - This in an approximation...
   def isTaskKey[T](key: sbt.ScopedKey[T]): Boolean = {
     val mf = key.key.manifest
-    mf.erasure == TaskClass
+    mf.runtimeClass == TaskClass
```

Comment:
I guess we dropped scala 2.9?  I remember we were forced to use erasure for a time (maybe that was 2.10, not sure), maybe that's gone.  If the build starts passing, then cool, otehrwise we may have to wait on this one.